src/spring-cloud/azext_spring_cloud/azext_metadata.json

```diff
@@ -1,4 +1,4 @@
 {
     "azext.isPreview": false,
-    "azext.minCliCoreVersion": "2.0.67"
+    "azext.minCliCoreVersion": "2.25.0"

```

Comment:
just curious, why the minor version changes from 0.x => 25.x.

Reply:
Since `connection_string` is not available from data model in old version until `2.25.0`,
after upgrade to `2.25.0`, you can access that from model:
`from azure.mgmt.applicationinsights import ApplicationInsightsManagementClient`